src/components/past-events/past-events.jsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import classes from './past-events.less';
+
+const DiamondAngle = '26deg';
```

Comment:
well I can replace it on var, I use it in different elements... so I need something like var)